library/src/main/java/com/meisolsson/githubsdk/model/IssueEvent.java

```diff
@@ -73,6 +91,22 @@
     @Nullable
     public abstract User actor();
 
+    @Json(name = "review_requester")
+    @Nullable
+    public abstract User reviewRequester();
+
+    @Json(name = "requested_reviewer")
+    @Nullable
+    public abstract User requestedReviewer();
+
+    @Json(name = "requested_reviewers")
+    @Nullable
+    public abstract List<User> requestedReviewers();
+
+    @Json(name = "dismissed_review")
+    @Nullable
+    public abstract DismissedReview dismissedReview();
+
```

Comment:
Missing for the timeline API:
- for 'reviewed': state, submitted_at
- for 'reviewed' and 'commented': author_association
- for 'committed': author, committer, tree (sha/url), message, parents (list of sha/url/html_url), verification
- for 'cross-referenced: source (type/issue)